go.mod

```diff
@@ -106,7 +107,7 @@ require (
 	github.com/spf13/jwalterweatherman v0.0.0-20170901151539-12bd96e66386 // indirect
 	github.com/spf13/pflag v1.0.1
 	github.com/spf13/viper v1.0.1-0.20171207042631-1a0c4a370c3e
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.4.0
```

Comment:
What is the reason to upgrade this library?

Reply:
I think while updating the go-chef vendor library, have found minor update in testify lib as well, so updated in this PR only.